src/install/python/python-frameworks/python-agent-paste.mdx

```diff
@@ -0,0 +1,38 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using Paste
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command.
+
+### Admin script integration method (recommended) [#integrated-server]
```

Comment:
Remove this section.

Reply:
@hmstepanek I removed this section: "Admin script integration method."